plugins/main/bridge/bridge_test.go

```diff
@@ -17,6 +17,7 @@ package main
 import (
 	"encoding/json"
 	"fmt"
+	"io/ioutil"
```

Comment:
Package `io/ioutil` is marked deprecated since Go 1.16. It would be better to use `os` package.

Reply:
removed, thanks